components/dropdownlist/overview.md

```diff
@@ -116,41 +131,17 @@ The DropDownList provides the following features:
 @code {
     protected List<string> MyStringList = new List<string>() { "first", "second", "third" };
 
+    //Current value is null (no item is sellected) which allows the DefaultText to be displayed.
     protected string MyStringItem { get; set; }
 
     protected List<int> MyIntList = new List<int>() { 1, 2, 3 };
 
+    //Current value is 0 (no item is sellected) which allows the DefaultText to be displayed.
     protected int MyIntItem { get; set; }
 }
 ````
 
 
-
```

Comment:
Indeed, this should be removed. I don't even remember why I had to add it. Maybe we've had a bug at some point.